profiles/nis/REQUIREMENTS

```diff
@@ -4,6 +4,10 @@ Make sure that NIS service is configured and enabled. See NIS documentation for
                                                                                           {include if "with-pam-u2f"}
 - with-pam-u2f is selected, make sure that the pam u2f module is installed                {include if "with-pam-u2f"}
   - users can then configure keys using the pamu2fcfg tool                                {include if "with-pam-u2f"}
+
```

Comment:
Remove this line.

Reply:
The blank line have been removed  I believe this resolves the comment.